Yosemite/Yosemite/Stores/AppSettingsStore.swift

```diff
@@ -619,6 +638,57 @@ private extension AppSettingsStore {
     }
 }
 
+// MARK: - Orders Settings
+//
+private extension AppSettingsStore {
+    func loadOrdersSettings(siteID: Int64, onCompletion: (Result<StoredOrderSettings.Setting, Error>) -> Void) {
+        guard let allSavedSettings: StoredOrderSettings = try? fileStorage.data(for: ordersSettingsURL) else {
+            let error = AppSettingsStoreErrors.noOrdersSettings
+            onCompletion(.failure(error))
+            return
+        }
+
+        guard let settingsUnwrapped = allSavedSettings.settings[siteID] else {
```

Comment:
nit: Is it worth chaining the optionals since the errors are the same?

```suggestion
        guard let allSavedSettings: StoredOrderSettings = try? fileStorage.data(for: ordersSettingsURL),
          let settingsUnwrapped = allSavedSettings.settings[siteID] else {
```